my-dsl.md

```diff
@@ -18,21 +35,78 @@ _Fowler writes about a spectrum of languages, from general-purpose languages to
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
+The Vimscript language is certainly closer to the "purely" domain-specific
+side of the spectrum than the general-purpose languages side. This is because
+the _vimrc_ commands only expose a very limited subset of commands with the
+sole intention of customizing a Vim user's experience. 
+[Here](http://vim.wikia.com/wiki/Example_vimrc) you can see an example of 
+a _.vimrc_ file. Notice how there appears to be no functionality for a complex
+non-domain specific task. Although constructs such as loops do exists, they are
+a more advance feature frequently used with macros. Similarly the extensive
+syntax highlighting scripts have a limited use beyond customizing text display.
+An example of a syntax script can be seen
+[here](http://vim.wikia.com/wiki/Creating_your_own_syntax_files).
+
+Although some developers have done some really funny and clever things using
+Vimscript and their Vim customization files, including some plugins that
+replicate an entire IDE in text form, the scope of these commands is greatly
+limited and focused to extending and customizing a text editor.
+
+That being said, someone has made a stand alone game to teach Vim where the
+controls and puzzles are all related to Vim's keys and commands. I guess that
+means this language can be used beyond the scope of editor customization, but
+that's the joke of the game, not a _real_ use case. Check it out
+[here](http://vim-adventures.com/).
```

Comment:
Since the text mentions that configuration files are not DSLs, I think it would be helpful to take specific quotes from the text to argue why Vimscript expands past what they are explaining. It would be nice to get examples of how Vimscript can be used for more than just setting text size and highlighting color. Can the different parts of the language be used together to produce new results? In the example you gave, it seemed like all the tasks just involved setting a constant value. I don't really see how this is a language that allows for complex ideas to be expressed using mixtures of the language's nouns and verbs.